swagger2.0/oic.sec.cred.swagger.json

```diff
@@ -246,7 +245,7 @@
                 "type": "integer"
               },
               "credtype": {
-                 "description": "Representation of this credential's type\nCredential Types - Cred type encoded as a bitmask.0 - Empty credential used for testing1 - Symmetric pair-wise key2 - Symmetric group key4 - Asymmetric signing key8 - Asymmetric signing key with certificate16 - PIN or password32 - Asymmetric encryption key.",
+                 "description": "Represents this credential’s type.\n0 – Used for testing\n1 – Symmetric pair-wise key\n2 – Symmetric group key\n4 – Asymmetric signing key\n8 – Asymmetric signing key with certificate\n16 – PIN or password\n32 – Asymmetric encryption key\n64 - Directly Provisioned OSCORE Security Context",
                  "maximum": 63,
```

Comment:
Don't we have to change the maximum value to 127 to allow for the 64s bit (i.e. 7th bit) to be set?